### DISCUSSION TITLE
How to move my microphone to a different buss

### DISCUSSION THREAD

**Discussion by Mudb0y** [onyx-and-iris/nvda-voicemeeter#18] [General]
I'd like to set up a system where I can patch through my applications to voice chat platforms such as Discord along with my microphone in the same audio device, but I'd like not to have my mic looped-back to me. I can't figure out the patch bay so any help would be appreciated.

**Comment by onyx-and-iris** [comment]
Hello. It sounds like you simply need to route your applications + mic through a single mixbus and then disable your mic for your physical output. That way, discord users can hear your application + mic but you won't hear your mic.

If you need a detailed walkthrough on bus assignments then the Voicemeeter discord is probably the best place to ask.
https://discord.gg/vb-audio-755690270795890739

I hope that helps.
